dts/bindings/gpio/arm,mmio32-gpio.yaml

```diff
@@ -1,6 +1,6 @@
-description: GPIO controller on ARM MPS2 FPGA
+description: ARM MMIO32 GPIO
 
-compatible: "arm,mps2-fpgaio-gpio"
+compatible: "arm,mmio32-gpio"
```

Comment:
There shouldn't be anything Arm-specific about this driver - while we are at it, how about renaming it to something without "arm" in it? The Linux kernel uses "basic-mmio-gpio" (see https://github.com/torvalds/linux/blob/master/drivers/gpio/gpio-mmio.c).

Reply:
As far as I am aware this is only used on the mps2/mps3 based boards.
It’s been tailored to the limitations of these peripherals and differs from basic-mmio-gpio in a few ways (eg, it does not support changing the direction of a pin).

Are you aware of anyother device that may benefit from it?

Reply:
Ok, fair enough.